easyInterface/Diffraction/Interface.py

```diff
@@ -339,6 +341,8 @@ def updatePhases(self):
             self.project_dict.endBulkUpdate()
         else:
             k, v = self.project_dict['phases'].dictComparison(phases)
+            self._log.debug(k)
```

Comment:
Log should not be used this way. I'd suggest removal

Reply:
Debug logging is removed